types/gatherer.d.ts

```diff
@@ -50,6 +51,7 @@ declare module Gatherer {
       on(event: 'protocolevent', callback: (payload: Protocol.RawEventMessage) => void): void
       off(event: 'protocolevent', callback: (payload: Protocol.RawEventMessage) => void): void
     };
+    networkMonitor: NetworkMonitor;
```

Comment:
I know I should type [just the public interface here](https://github.com/GoogleChrome/lighthouse/actions/runs/4929932153/jobs/8810214165?pr=15055#step:12:145), but then we get this error...

<img width="1197" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/4071474/df9c0d2b-249b-49f9-bf03-f0e060399f2a">

How can we resolve this?

Reply:
Should we pull `waitForFullyLoaded` (and whatever else...) into NetworkMonitor? expose as `NetworkMonitor.waitForFullyLoaded(session, opts)`?

Reply:
Eh, `waitForFullyLoaded` does other non-network stuff like CPU idle. I think it's fine this way personally.